0124-securing-iot-governance-through-staking-rewards.md

```diff
@@ -1,4 +1,4 @@
-# HIP 124: Securing IOT Governance Through Staking Rewards
+# HIP 124: Securing IOT Governance Through Voting Rewards
```

Comment:
file needs a rename. @hiptron can manage that on merge.

Reply:
what needs to be done, just rename the file?

Reply:
rename the file yeah. please